github/repos_traffic.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2013 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+)
+
+// Referrer represent a referrer information
+type Referrer struct {
+	Referrer *string `json:"referrer,omitempty"`
+	Count    *int    `json:"count,omitempty"`
+	Uniques  *int    `json:"uniques,omitempty"`
+}
+
+// Path represent a referrer information
+type Path struct {
+	Path    *string `json:"path,omitempty"`
+	Title   *string `json:"title,omitempty"`
+	Count   *int    `json:"count,omitempty"`
+	Uniques *int    `json:"uniques,omitempty"`
+}
+
+// Time represents a time stamp as in a datapoint
+//
+// This is necessary becausegithub uses unix timestamp and not YYYY-MM-DDTHH:MM:SSZ
+type Time struct {
+	time.Time
+}
```

Comment:
Although I guess this approach also causes problems with times before 1970-01-12 that are expressed in milliseconds (since they'd be interpreted as seconds and be about 30 years off).  Not sure how likely it is to have dates in that range in the traffic API though.  

Another option, which is *much* cleaner, but requires a bit more work is to support some basic tag options.  So you could have:

```go
struct {
    Timestamp *Timestamp `json:"timestamp,omitempty" precision:"millisecond"`
}
```

or something.  Though I'm pretty sure you don't have access to the struct tags inside of `Timestamp.UnmarshalJSON`, so a lot would have to be reworked.  Almost certainly not worth the trouble.